docs/src/pages/de/quick-start.md

```diff
@@ -30,46 +30,46 @@ npm run build
 
 Um mehr über den Installationsprozess und den ersten Gebrauch von Astro zu lernen, [lies unseren Installations-Leitfaden.](de/installation)
 
-Wenn du es bevorzugst anhand eines Beispiels zu lernen, schau dir die [komplette Liste an Beispielen](https://github.com/withastro/astro/tree/main/examples) auf GitHub an.
-Du kannst jedes Beispiel ausprobieren, indem du `npm init astro -- --template "EXAMPLE_NAME"` ausführst.
+Wenn du es bevorzugst, anhand eines Beispiels zu lernen, schau Dir die [komplette Liste an Beispielen](https://github.com/withastro/astro/tree/main/examples) auf GitHub an.
```

Comment:
I believe only `Sie` gets capitalized; `dir`, `deine`, etc. do not. I don’t think most of these are typos.

My German is too bad to verify the other word changes, though—they may still be valid. Would love a review from someone fluent.

Reply:
Oops! Missed @borisv’s comment above.

I’m going to close this PR in favor of #1830 to cut down on possible confusion (please move any suggestions for change to that PR!)